cav_msgs/msg/LaneChangeStatus.msg

```diff
@@ -8,8 +8,12 @@ uint8 status
 
 # enumeration values for status:
 uint8   PLAN_SENT=1
-uint8   ACCEPTANCE_RECEIVED=2
-uint8   REJECTION_RECEIVED=3
-uint8   OTHER_RECEIVED=4
-uint8   PLANNING_SUCCESS=5
-uint8   TIMED_OUT=6
+uint8   REQUEST_RECEIVED=2
```

Comment:
# status variables and its value is from the enum
uint8 status

# status description. It is used to describe the status with more details
string description

Is it possible to add description? so that the UI do not need to add additional description and display it? It is a suggestion since UI might miss interpret (description) the meaning of each status. It  would be nice that UI can get this description dynamically from backend.